index.html

```diff
@@ -584,23 +584,18 @@ <h2 class='informative'>Implementation Patterns</h2>
       <h2>Resource Authorization</h2>
       <p>
         To configure access control restrictions, implementations MUST follow the recommendations
-        of [[!SOLIDWEBAC]].
+        of [[!SOLIDWEBAC]] with the following additional requirements:
       </p>
-      <p>
-        Implementations supporting access control MUST use the [[!WEBAC]] model to
-        configure that functionality. Any access control resource in use MUST be a LDPRS,
-        which SHOULD be located in the same server as the controlled resource. Implementations
-        MUST support extended control relationships in which the access control resource for some
-        resource has its own access control resource and so on. Implementations MUST make access
-        control resources discoverable via the <code>Link@rel="acl"</code> HTTP header, as discussed
-        <a href="https://github.com/solid/web-access-control-spec#acl-resource-location-discovery">here</a>.
-      </p>
-
+      <section id="solid-ldp-acls">
+        <h2>ACLs are LDP RDF Sources</h2>
+        <p>
+          The linked Access Control List Resource (ACL) for a controlled resource by a conforming server MUST itself be a LDPRS. This ACL resource SHOULD be located in the same server as the controlled resource.
```

Comment:
nit picking, but it's nice to wrap the lines at some sensible length to make reading them easier. (There are other advantages, too!)